### DISCUSSION TITLE
What is the total cost of using extpay

### DISCUSSION THREAD

**Discussion by Shashank003** [Glench/ExtPay#99] [Q&A] [answered]
You have mentioned you take 5 percent of every transaction. Is that 5 percent added to stripe percentage, making it 7.9 percent total, or is it 5 percent total( stripe takes 2.9, you take 2.1)

**Comment by Glench** [comment, marked as answer]
The cost of using ExtPay is 5%, in addition to the cost to use Stripe (2.9%).

**Reply by Shashank003** [reply]
A follow up question, yearly and monthly subscriptions are supported? I want to charge my customers monthly, and give them an option to pay for yearly access, and get a discount. Is that supported with extpay?

**Reply by Glench** [reply]
Not officially at the moment, but you can see a workaround here: https://github.com/Glench/ExtPay/discussions/22

**Reply by rurimiao** [reply]
Would you please explain the stripe fees more clearly? I notice that stripe has two modes, one is 2.9% + 30 cents for each payment, the other is micro-payments but maybe it is unapplicable for now. So for instance, what would happen if I set monthly subscription fee at $1, would the total cost before tax is `ExtPay 5% + Stripe 2.9% + Stripe 30 cents`? It is almost 37.9%.

**Reply by akump** [reply]
@rurimiao that's correct. the flat 30 cents adds up.
